src/components/filters/FilterSelect.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {Select} from "antd";
+import "antd/es/select/style/css";
+
+const { Option } = Select;
+
+const FilterSelect = ({
+  filterType,
+  filterTypeName,
+  options,
+  mode,
+  placeholder,
+  onChange,
+  filters
+}) => {
+  const convertFiltersToArr = (filters) => {
+    return typeof(filters) == 'string' ? filters.split(",") : filters
+  }
+
+  const handleChange = (val) => {
+    onChange(filterType, val)
+  }
+  return <>
+    <div>
+      {`${filterTypeName.toUpperCase()}: `}
+      <Select
+        id={`${filterType}Select`}
+        mode={mode}
+        style={{ width: 200 }}
+
+        placeholder={placeholder}
+        onChange={handleChange}
+        defaultValue={convertFiltersToArr(filters[filterType])}
+
+      >
```

Comment:
Above:
 - Remove blank lines
 - Not sure what you're gonna do with `id`, but if ever you need https://www.npmjs.com/package/cuid is nice for auto-generated IDs